### ISSUE TITLE
Missing translate="no" attribute on composer causes problems in Edge

### ISSUE THREAD

**Issue by benparsons** [vector-im/element-web#25624]
#### What would you like to do?

Users report that when they use Edge automatic translation, the composer starts behaving unexpectedly - typing will no longer show characters.

#### Why would you like to do it?

They want to ignore input into the composer while in a translate-mode.

#### How would you like to achieve it?

Adding translate="no" onto the composer element should solve the problem.

### Have you considered any alternatives?

Having the composer be an input field apparently does this automatically, but the attribute works fine.

### Additional context

Less confidently, this also may happen in Chrome, but only really-well tested in Edge.
